src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -132,55 +155,67 @@ class PillSelector extends Component {
             inputProps,
             onInput,
             onRemove,
+            onSuggestedPillAdd,
             placeholder,
             selectedOptions,
+            suggestedPillsData,
+            suggestedPillsTitle,
             ...rest
         } = this.props;
+        const suggestedPillsEnabled = suggestedPillsData && suggestedPillsData.length > 0;
         const classes = classNames('pill-selector-input-wrapper', {
             'is-disabled': disabled,
             'is-focused': isFocused,
             'show-error': !!error,
+            'pill-selector-suggestions-enabled': suggestedPillsEnabled,
         });
 
         return (
-            <Tooltip isShown={!!error} position="middle-right" text={error} theme="error">
-                {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
+            <Tooltip isShown={!!error} text={error || ''} position="middle-right" theme="error">
                 <span
                     className={classes}
                     onBlur={this.handleBlur}
                     onClick={this.handleClick}
                     onFocus={this.handleFocus}
                     onKeyDown={this.handleKeyDown}
+                    role="button"
+                    tabIndex={0}
                 >
                     {selectedOptions.map((option, index) => (
                         <Pill
-                            key={option.value}
                             isSelected={index === selectedIndex}
+                            key={option.value}
                             onRemove={onRemove.bind(this, option, index)}
                             text={option.text}
                         />
                     ))}
                     {/* hidden element for focus/key events during pill selection */}
                     <span
-                        ref={this.hiddenRef}
                         aria-hidden="true"
                         className="accessibility-hidden"
                         onBlur={this.resetSelectedIndex}
+                        ref={this.hiddenRef}
                         tabIndex={-1}
                     />
                     <input
                         {...rest}
                         {...inputProps}
-                        ref={input => {
-                            this.inputEl = input;
-                        }}
                         autoComplete="off"
                         className={classNames('pill-selector-input', className)}
                         disabled={disabled}
                         onInput={onInput}
                         placeholder={this.getNumSelected() === 0 ? placeholder : ''}
+                        ref={input => {
+                            this.inputEl = input;
+                        }}
                         type="text"
                     />
+                    <SuggestedPillsRow
```

Comment:
Is this component to be added ONLY if `suggestedPillsEnabled` is `true`?

If so
```
{suggestedPillsEnabled && <SuggestedPillsRow foo={foo} bar={bar} />}
```

Reply:
`SuggestedPillsRow` only renders when `suggestedPillsData` has length of greater than 0 and the ids of the items are not included in the form. I'm open to adding that prop, but I don't think it's totally necessary since I believe it is implicit in the data.